tests_trung/testcases/da_mp/da_mp_test.py

```diff
@@ -0,0 +1,113 @@
+        self.manager_page.add_new_page(self.page_2, self.page_1)
+
+        logging.info("Verify that Page 2 is added")
+        assert self.page_2 in browser.get_title()
+
+        logging.info("3. Select page 1")
+        self.manager_page.select_page(self.page_1)
+
+        logging.info("4. Edit name Page 1 to Page 3")
+        self.manager_page.edit_page(self.page_3)
+
+        logging.info("Verify that Page 3 is updated")
+        assert self.page_3 in browser.get_title()
+
+        logging.info("5. Select Page 2")
+        self.manager_page.select_page(
+            "{0}>{1}".format(self.page_3, self.page_2))
+
+        logging.info("6. Edit name Page 2 to Page 4")
+        self.manager_page.edit_page(self.page_4)
+
+        logging.info("Verify that Page 4 is updated")
+        assert self.page_4 in browser.get_title()
+
+    @pytest.mark.usefixtures("clean_up_tc022")
+    def test_da_mp_tc022(self):
+
+        logging.info("1. Add new page Test")
+        self.manager_page.add_new_page(self.test_page, constants.ROOT_PAGE)
+
+        logging.info("Verify that Test is added")
+        assert self.test_page in browser.get_title()
+
+        logging.info("2. Add new page TestChild1 is child of Test page")
+        self.manager_page.add_new_page(self.test_child_page, self.test_page)
+
+        logging.info("Verify that TestChild1 is added")
+        assert self.test_child_page in browser.get_title()
+
+        logging.info("3. Add new page TestChild1 is child of Test page")
+        self.manager_page.select_global_setting_option("Add Page")
+        self.manager_page.enter_new_page_infor(
+            self.test_child_page, self.test_page)
+        self.manager_page.click_ok()
+
+        logging.info(
+            "Verify that warning message 'TestChild1 already exist. Please enter a diffrerent name.' appears")
+        assert "{0} already exists. Please enter a different name.".format(
+            self.test_child_page) in browser.get_alert_text()
+
+    @pytest.fixture()
+    def clean_up_tc021(self):
+
+        logging.info("Navigate to Dashboard")
+        self.login_page.open_DA_login()
+
+        logging.info("Login DA with valid user")
+        self.login_page.login()
+
+        yield
+
+        logging.info("Delete the created pages")
+        self.manager_page.select_page(
+            "{0}>{1}".format(self.page_3, self.page_4))
+        self.manager_page.delete_page()
+        self.manager_page.select_page(self.page_3)
+        self.manager_page.delete_page()
+
+    @pytest.fixture()
+    def clean_up_tc022(self):
+
+        yield
+
+        browser.accept_alert()
+        self.manager_page.click_cancel()
+        logging.info("Delete the created pages")
+        self.manager_page.select_page("{0}>{1}".format(
+            self.test_page, self.test_child_page))
+        self.manager_page.delete_page()
+        self.manager_page.select_page(self.test_page)
+        self.manager_page.delete_page()
```

Comment:
Should delete page by name instead of calling 2 methods select page and then delete page. 
`self.manager_page.delete_page(self.test_page)`